### PR TITLE
Sed 3042 decommissioned legacy dashboards

### PR DESCRIPTION
**Pull request by lucianbaciu**

### CODE REVIEW

File: projects/step-frontend/src/lib/modules/timeseries/modules/chart/components/chart-skeleton/chart-skeleton.component.html [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<div class="chart-skeleton">
+<div class="chart-skeleton" [style.height]="height + 'px'">
```
Comment (dvladir) [inline projects/step-frontend/src/lib/modules/timeseries/modules/chart/components/chart-skeleton/chart-skeleton.component.html:1]:
```suggestion
<div class="chart-skeleton" [style.height.px]="height">
```

File: projects/step-frontend/src/lib/modules/execution/components/dashlet-execution-viz/dashlet-execution-viz.component.html [hunk 2]
```diff
@@ -1,2 +1 @@
-<step-execution-performance [executionId]="_state.executionId!" [executionInput]="_state.execution">
-</step-execution-performance>
+<step-execution-page [execution]="_state.execution!"> </step-execution-page>
```
Comment (dvladir) [inline projects/step-frontend/src/lib/modules/execution/components/dashlet-execution-viz/dashlet-execution-viz.component.html:1]:
```suggestion
<step-execution-page [execution]="_state.execution!"/>
```

File: projects/step-frontend/src/lib/modules/timeseries/modules/filter-bar/components/dashboard-filter-bar/dashboard-filter-bar.component.ts [hunk 3]
```diff
@@ -273,7 +273,7 @@ export class DashboardFilterBarComponent implements OnInit, OnDestroy {
     if (existingItems.length > 1) {
       // the filter is duplicated
       this._snackbar.open('Filter not applied', 'dismiss');
-      this.removeFilterItem(index);
+      // this.removeFilterItem(index);
```
Comment (dvladir) [inline projects/step-frontend/src/lib/modules/timeseries/modules/filter-bar/components/dashboard-filter-bar/dashboard-filter-bar.component.ts:276]: Should it be removed?

File: projects/step-frontend/src/lib/modules/timeseries/modules/legacy/components/execution-page/ts-execution-page.component.html [hunk 4]
```diff
@@ -1,38 +1,37 @@
...
+  <!--    <step-ts-resolution-picker (resolutionChange)="handleResolutionChange($event)"></step-ts-resolution-picker>-->
+  <!--    <button mat-menu-item (click)="toggleCompareMode()">-->
+  <!--      <step-icon name="bar-chart-2" [class.active]="compareModeEnabled"></step-icon>-->
+  <!--      {{ compareModeEnabled ? 'Disable Compare Mode' : 'Enable Compare Mode' }}-->
+  <!--    </button>-->
+  <!--    <button mat-menu-item (click)="exportRawData()">-->
+  <!--      <step-icon name="download"></step-icon>-->
+  <!--      <span>Export Raw Measurements</span>-->
+  <!--    </button>-->
+  <!--  </mat-menu>-->
```
Comment (dvladir) [inline projects/step-frontend/src/lib/modules/timeseries/modules/legacy/components/execution-page/ts-execution-page.component.html:23]: Is this part relevant?

**Review by dvladir — CHANGES_REQUESTED** [review]

File: projects/step-frontend/src/lib/modules/timeseries/modules/legacy/components/execution-page/ts-execution-page.component.html (hunk 4 above)
Comment (lucianbaciu) [reply projects/step-frontend/src/lib/modules/timeseries/modules/legacy/components/execution-page/ts-execution-page.component.html:23]: no, it's legacy. will be completely removed

**Pull request closed without merging**